### PR TITLE
Initial branch commit

### PR DESCRIPTION
**Pull request by o-wrobel**
I've added additional data to README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,4 @@
-# mins
+# *Minesweeper*
```
Comment (BlaMarDev) [inline README.md]: You should change italic style. Remove it.

**Comment by GamerGod99** [comment]
test

File: README.md (hunk 1 above)
Comment (o-wrobel) [reply README.md]: Done. Replaced with bold

**Review by BlaMarDev — COMMENTED** [review]
ok

**Pull request merged**